PyU4V/replication.py

```diff
@@ -46,13 +46,17 @@
 GENERATION = constants.GENERATION
 RDFG = constants.RDFG
 VOLUME = constants.VOLUME
+RDF_DIRECTOR = constants.RDF_DIRECTOR
+PORT = constants.PORT
+REMOTE_PORT = constants.REMOTE_PORT
+RDF_GROUP = constants.RDF_GROUP
 
 
 class ReplicationFunctions(object):
-    """ReplicationFunctions."""
+    '''ReplicationFunctions.'''
```

Comment:
Looks like they have been changed throughout the document.  Please change them to ".  Thanks

Reply:
Thanks Helen that was a find replace error.